### PR TITLE
Update to use typings, and sort activities in the correct order

### PR DESCRIPTION
**Pull request by obvioussean**

### CODE REVIEW

File: scripts/observer.ts [hunk 1]
```diff
@@ -191,17 +190,23 @@ export class ActivityManager {
...
         if (existingEntry) {
+            // remove the entry from it's current location
+            // since we're going to bring to the top of the list.
             activities.splice(existingEntryIndex, 1);
         }
 
-        activities.splice(1000);
-        activities.push(activity);
+        // put the activity on the top of the array
+        activities.unshift(activity);
```
Comment (mmanela) [inline scripts/observer.ts:215]: Doesn't this mean that the list on MSENG will be reverse ordered and the we start putting new entries in the other order?

Comment (obvioussean) [inline scripts/observer.ts:215]: Yes.  I'm fine with it being in a weird state for mseng while people start generating new activities.

Comment (obvioussean) [inline scripts/observer.ts:215]: I updated to handle this scenario

**Comment by mmanela** [comment]
Thanks!

**Pull request merged**